src/api/decommission-service/helpers/trigger-remove-workflows.js

```diff
@@ -18,22 +18,21 @@ async function triggerRemoveWorkflows(serviceName, repository, logger) {
   logger.info(`Triggering remove workflows service: ${serviceName}`)
   const isTestSuite = repository.topics.includes('test-suite')
 
-  if (isFeatureEnabled(featureToggles.removeServiceWorkflows)) {
-    if (!isTestSuite) {
-      const zone = repository.topics.includes('backend')
-        ? 'Protected'
-        : 'Public'
+  if (!isFeatureEnabled(featureToggles.removeServiceWorkflows)) {
+    logger.info('Remove service workflows feature is disabled')
+    return
+  }
 
-      await removeDashboard(serviceName, logger)
-      await removeNginxUpstreams(serviceName, zone, logger)
-    }
+  if (!isTestSuite) {
+    const zone = repository.topics.includes('backend') ? 'Protected' : 'Public'
 
-    await removeAppConfig(serviceName, logger)
-    await removeSquidConfig(serviceName, logger)
-    await removeTenantInfrastructure(serviceName, logger)
-  } else {
-    logger.info('Remove service workflows feature is disabled')
+    await removeDashboard(serviceName, logger)
+    await removeNginxUpstreams(serviceName, zone, logger)
   }
+
+  await removeAppConfig(serviceName, logger)
```

Comment:
`Promise.all()` or `Promise.allSettled()` would mean these can become parallel

Reply:
Could do, but there is no need as this has no performance or concurrency requirements. In fact, as sequential as possible is preferred.